server/router/server.go

```diff
@@ -305,6 +305,16 @@ func NewRouter(_ context.Context, h models.HandlerInterface, port int, g http.Ha
 	gMux.Handle("/api/integrations/credentials", h.ProviderMiddleware(h.AuthMiddleware(h.SessionInjectorMiddleware(h.DeleteUserCredential), models.ProviderAuth))).
 		Methods("DELETE")
 
+	// Handlers for User Connections
+	gMux.Handle("/api/integrations/connection", h.ProviderMiddleware(h.AuthMiddleware(h.SessionInjectorMiddleware(h.SaveConnection), models.ProviderAuth))).
+		Methods("POST")
+	gMux.Handle("/api/integrations/connections/{connectionKind}", h.ProviderMiddleware(h.AuthMiddleware(h.SessionInjectorMiddleware(h.GetConnections), models.ProviderAuth))).
+		Methods("GET")
+	gMux.Handle("/api/integrations/connection/{connectionKind}/edit", h.ProviderMiddleware(h.AuthMiddleware(h.SessionInjectorMiddleware(h.UpdateConnection), models.ProviderAuth))).
```

Comment:
Do we really need `/edit` ? We can have the same endpoint with the `PUT` verb. Similarly for `/delete`

Reply:
> Do we really need /edit ?

We don't.

I think I kept it like that to keep 1:1 correlation with Meshery Cloud